### ISSUE TITLE
problem when installing with pip

### ISSUE THREAD

**Issue by quaquel** [SALib/SALib#279]
I keep running into a weird SSL error when trying to install SALib. Below is a screenshot with the error. This is on a brand new windows server 2019 machine with the latest anaconda 64 bit distribution. I have had the same on clean windows 10 machines as well. Any idea what is going on and how it might be fixed? 

I have tried various suggested fixes such as updating pip etc, but I have the most up to date versions here. My hunch is that it has to do with some configurations related to pyscaffold. 

<img width="1680" alt="Screenshot 2019-12-19 at 16 10 46" src="https://user-images.githubusercontent.com/1073249/71184590-46f46e00-227a-11ea-9f54-d41fb4e6f996.png">

**Comment by jdherman** [comment]
Weird indeed. Is this output from just running `pip install SALib` ? Does the same thing happen with other pip installs?

I haven't seen this before, so you've probably found the same info. But did you try this thing:

`pip install --trusted-host=pypi.org --trusted-host=files.pythonhosted.org --user {name of whatever I'm installing}`

from here: https://python-forum.io/Thread-Cannot-update-or-install-anything-with-pip-SSL-error

**Comment by quaquel** [comment]
I have tried that one and it did not work on either windows 10 or windows server 2019. 

To be sure, I completely disabled all firewall settings on windows 10 (I can't do this on server 2019, the university admins would shut me out immediately) but that did not help.

**Comment by jdherman** [comment]
Ok. Is this only happening for pip install salib? Does it still happen if you run `pip install pyscaffold` directly?

**Comment by quaquel** [comment]
I can install pyscaffold directly. But if I than retry SALib, it still fails with the same error.

**Comment by jdherman** [comment]
Strange. My current guess is either:

1. Something is wrong with the SSL packages, either the python packages or the system ones. Some of these installations discussed here might help.
https://github.com/pypa/pip/issues/4205

2. Or, something is wrong with the pyscaffold configuration specifically in SALib. (Is there any way to test if this also happens with other packages that are using pyscaffold?) This is definitely possible but I'm not sure where to start. 

If anyone else runs into this problem, or has an idea, let us know. Also I'd be interested if anyone can reproduce it on Windows 10.

Jan if you need a quick solution, you can just download the zip file and run it as a module. We should be able to figure this out though.

**Comment by quaquel** [comment]
I did something a little bit more involved:
I downloaded the zip file from GitHub for the latest stable release
copied the directory to site-packages
removed the setup_requires keyword argument from setup.py
ran ```python setup.py install```

this seems to make it work, but I still have no clue what is going on. It might be an issue with pyscaffold. If I read SALib 1.3.8 correctly, pyscaffold needs to be between 3.0 and 3.1 although pyscaffold is already up to 3.2.3.
